src/platforms/Gather/GatherConnectionForm.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  HStack,
+  Icon,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Link,
+  Stack,
+  Text,
+  Tooltip,
+} from "@chakra-ui/react"
+import { AddGatherFormType } from "components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddGatherPanel"
+import { Eye, EyeClosed, Question } from "phosphor-react"
+import { useState } from "react"
+import { useFormContext, useWatch } from "react-hook-form"
+import GatherConnectionStatusAlert from "./GatherConnectionStatusAlert"
+import useGatherAccess from "./hooks/useGatherAccess"
+
+const GatherConnectionForm = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useFormContext<AddGatherFormType>()
+
+  const [showApiKey, setShowApiKey] = useState(false)
+
+  const [gatherApiKey, gatherSpaceId] = useWatch({
+    name: ["gatherApiKey", "gatherSpaceId"],
+  })
+  const {
+    success: accessSuccess,
+    isLoading: accessLoading,
+    error: accessError,
+  } = useGatherAccess(
+    !errors?.gatherApiKey && gatherApiKey,
+    !errors?.gatherSpaceId && gatherSpaceId
+  )
+
+  return (
+    <>
+      <GatherConnectionStatusAlert
+        isLoading={accessLoading}
+        success={accessSuccess}
+        error={accessError}
+      />
+
+      <FormControl pt={{ md: 0.5 }} isInvalid={!!errors?.gatherApiKey}>
+        <Stack>
+          <HStack>
+            <Text as="span">API key:</Text>
+            <Tooltip
+              label="Used to securely communicate with Gather"
+              placement="top"
+              hasArrow
+            >
+              <Icon as={Question} color="GrayText" />
+            </Tooltip>
+          </HStack>
+          <InputGroup>
+            <Input
+              autoComplete="off"
+              type={showApiKey ? "text" : "password"}
```

Comment:
When being a pw and clicking next, the browser asked me if I'd want to save it, idk if we can prevent that

Reply:
The `autoComplete='off'` attribute should theoretically address this issue, yet it appears to be ineffective. Should we then default to a standard text input?

Most solutions to this often seem excessively complicated. We could apply the `-webkit-text-security: disc` property in CSS, which appears to be widely supported on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-security#browser_compatibility) and [Can I Use](https://caniuse.com/?search=-webkit-text-security), though it's still considered non-standard.